### PR TITLE
[HOLD] DEVELOPER-2830 Created CDK combined get started layout

### PR DESCRIPTION
**Pull request by adelasofia**

### CODE REVIEW

File: stylesheets/_get-started.scss [hunk 1]
```diff
@@ -272,3 +272,351 @@ ul.toc {
...
+        }
+      }
+      a {
+        font-size: 1em;
+      }
+    }
+  }
+}
+
+#get-started h2{
```
Comment (LightGuard) [inline stylesheets/_get-started.scss]: Using IDs, and to a lesser extend tag names, is often considered bad practice in css when you can get by with a class. I'v also found it difficult on the drupal side when the css heavily depends on the html structure to recreate things correctly.

Comment (adelasofia) [inline stylesheets/_get-started.scss]: Thanks, that class is on multiple pages so I'll create an Jira to fix this

**Comment by LightGuard** [comment]
@adelasofia Are we going to merge this and make the CSS changes in a different PR?

**Comment by adelasofia** [comment]
@LightGuard Yes, but this PR is still not ready to be merged

**Comment by LightGuard** [comment]
retest this please

**Comment by LightGuard** [comment]
This is blocking #990 from being merged

**Comment by LightGuard** [comment]
Disregard my last comment.

**Comment by adelasofia** [comment]
retest this please

**Comment by LightGuard** [comment]
Looks like we have failing acceptance tests @adelasofia, @RubyGemTSL any thoughts?

**Comment by RubyGemTSL** [comment]
@adelasofia @LightGuard

There's something not right with login page. Keycloak was either down or they have made changes to that page (that I was unaware of). Will retest and investigate.

**Comment by RubyGemTSL** [comment]
retest this please

**Comment by LightGuard** [comment]
@adelasofia Are we waiting for a sign off (and apparently now a merge conflict)?

**Comment by adelasofia** [comment]
@LightGuard Yes this is still work in progress

**Comment by adelasofia** [comment]
Approved by Mike Guerette and Rob Terzi

**Pull request merged**